CODEOWNERS

```diff
@@ -0,0 +1,39 @@
+## Dialects, Conversions, Tools
+
+# Arc
+**/Dialect/Arc @fabianschuiki
+tools/arcilator @fabianschuiki
+
+# ESI
+**/Dialect/ESI @teqdruid
+
+# FIRRTL
+**/FIRRTL* @darthscsi @dtzSiFive @seldridge
+*/firtool @dtzSiFive
+libs/Firtool @dtzSiFive
+
+# LLHD
+**/LLHD* @fabianschuiki
+tools/llhd-sim @fabianschuiki
+
+# Handshake
+**/Handshake* @hanchenye @stephenneuendorffer
```

Comment:
This should probably be @mortbopet @Dinistro @RamirezLucas now

Reply:
Great, thanks. Done!